code/game/objects/items/cigs_lighters.dm

```diff
@@ -330,9 +330,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	desc = "A Carp Classic brand cigarette. A small label on its side indicates that it does NOT contain carpotoxin."
 
 /obj/item/clothing/mask/cigarette/carp/Initialize()
-	. = ..()
 	if(prob(5))
 		list_reagents = list(/datum/reagent/drug/nicotine = 15, /datum/reagent/toxin/carpotoxin = 3) //they lied
+	. = ..()

```

Comment:
return ..()

Reply:
Went with Cobby's suggestions as they make sense to me.